docs/docs/gatsby-style-guide.md

```diff
@@ -278,3 +289,156 @@ The Hemingway App will assign a “grade level” for your writing. You should a
+  - start coding immediately without being an expert
+  - start a new project
+  - make edits and see them thru hot reloading
+  - publish a site
+  - do basic tasks like create pages, link between pages, create routing, change styles
+- Know how to and actually start and deploy a site as quickly as possible.
+- Be able to share their site.
+- Know how to and actually find more advanced tutorials and docs.
+- Have fun!
+- Use enough React to do basic tasks like creating pages, links, styles.
+
+### Tutorial tone and style
+
+The tone and style of the tutorial should effectively help the audience reach their goals.
+
+### Use personal "you" and be warm
+
+The main tutorial ought to use the same personal “you” like the rest of the docs; in addition, the tutorial ought to use a warm, validating tone by congratulating users, complimenting them, and generally saying things like “yay!” more often.
+
+> Why not use "yay" in the docs as well? Since the tutorial's goal is to help users complete a series of steps, it is possible and helpful to congratulate them on successfully completing each step. Guides in the docs act as reference guides that users can browse at will rather than read from top to bottom. It doesn't make sense to congratulate someone on finishing a guide, since it's not a series of steps.
+
+#### Don't make users think more than is necessary
+
+Because the audience of the tutorial is people who do not consider themselves experts in React, it's important to reduce the amount of new information to bare minimum. The goal: give people only the information necessary to complete a task and to know how to repeat the task again, outside of the context of the tutorial.
+
+In practice, you can reach this goal by two rules of thumb:
+
+- Reduce the number of hyperlinks, tabs, and environments to the least number required to complete the tasks in the tutorial.
+- When there are multiple ways to complete a task, give people only one way. This way ought to be the best practice possible within the constraints of the lowest supported versions of software. If the best practice isn't possible with the lowest supported versions of software, mention that as a side note.
+
+## Docs
+
+### Docs audience
+
+Through research, it's clear that developers of all skill levels read the docs and find them useful.
+
+The tutorial should focus on helping users with the following attributes and goals.
+
+Attributes:
+
+- intermediate to advanced at React
+- front-end developer
+- prefer using google search and/or `ctrl + f` to find things on the gatsbyjs.org site
+
+Looking for:
+
+- way to get a site up and running quickly
+- a quick way to get the right words, types, defaults, descriptions, parameters,and returns for the API
+- bits of source code to study and/or copy
+- step-by-step tutorials for advanced tasks
+- understanding how Gatsby works at a deep level, so deep that they could actually modify or customize their local stuff, or contribute to Gatsby core
```

Comment:
what do you mean by "local stuff"?

Reply:
haha no idea. I wrote that a while ago. Hm...probably "their own projects" is what I meant.

Reply:
probably meant "own projects"